pkg/environment/environment.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2020 IBM Corp.
+// SPDX-License-Identifier: Apache-2.0
+
+package environment
+
+import (
+	"os"
+)
+
+// Attributes that are defined in a config map or the runtime environment
+const (
+	UseTLS                string = "USE_TLS"
+	UseMTLS               string = "USE_MTLS"
+	CertSecretName        string = "CERT_SECRET_NAME"
+	CertSecretNamespace   string = "CERT_SECRET_NAMESPACE"
+	CACERTSecretName      string = "CACERT_SECRET_NAME"      //nolint:gosec
+	CACERTSecretNamespace string = "CACERT_SECRET_NAMESPACE" //nolint:gosec
+)
+
+// IsUsingTLS returns true if the connector communication should use tls.
+func IsUsingTLS() bool {
+	return os.Getenv(UseTLS) == "true"
```

Comment:
in order to cache the cases like "True" or "TRUE", can we add `strings.ToLower`

Reply:
Fixed. Thanks